test/androidTest/src/org/apache/cordova/test/CordovaPluginTest.java

```diff
@@ -0,0 +1,126 @@
+
+package org.apache.cordova.test;
+
+/*
```

Comment:
Whoops!  I used one of the other tests as a template - the package is above the copyright in all of them.  I'll move it regardless.